html/getstarted.html

```diff
@@ -0,0 +1,485 @@
+				<span class="nav-sub" onclick="ScrollTo('page-compiling')">Compiling manually</span><br>
+				<span class="nav-sub" style="padding-left:16px" onclick="ScrollTo('page-compiling-linux')">Linux</span><br>
+				<span class="nav-sub" style="padding-left:16px" onclick="ScrollTo('page-compiling-windows')">Windows</span><br>
+			</p>
+		</div>
+		<!-- SUB-NAVIGATION END -->
+
+		<div id="helper"></div>
+
+		<h3 style="text-align:left">Quick Start</h3>
+		<p align="justify">
+			There are multiple ways to get mlpack up and running. Python bindings can be
+			installed using pip or conda, or built from source (see <a id="built-source" href="https://www.mlpack.org/doc/mlpack-3.4.1/doxygen/build.html">Build from
+			source tutorials</a>). Julia bindings can be installed via Julia's 'Pkg' package manager. For C++, if mlpack is not available via your preferred
+			OS package manager, or if you need to build your own version (e.g. to apply
+			optimizations, use a different set of BLAS/LAPACK, or build a different
+			configuration), please refer to the <a id="built-manually" href="https://www.mlpack.org/doc/mlpack-3.4.1/doxygen/build.html">Build from
+			source</a> tutorials. For Windows, prebuilt binaries will help you get started
+			without the need of building mlpack. These packages include both the C++
+			mlpack library as well as the CLI tools.
+		</p>
+
+		<p align="justify">
+				Once you get mlpack running, check out the <a
+				href="https://www.mlpack.org/docs.html">documentation</a> or the <a
+				href="https://github.com/mlpack/examples/">examples repository</a>, which
+				contains simple example usages of mlpack.
+		</p>
+	</div>
+</center>
+<!-- QUICKSTART END -->
+
+<br>
+
+<!-- QUICKSTART-CLOUD START -->
+<div class="grid-simple-package">
+	<div></div>
+
+	<div style="width:200px;height:40px;border-radius:5px;background-color:#BABFC6">
+		<div style="padding-top:10px"><a style="text-decoration:none;color:black" href="http://lab.mlpack.org/v2/gh/mlpack/examples/master?urlpath=lab">Jupyter cloud</a></div>
+	</div>
+
+	<div style="width:200px;height:40px;border-radius:5px;background-color:#BABFC6">
+		<div style="padding-top:10px"><a style="text-decoration:none;color:black" href="">Docker Linux</a></div>
+	</div>
+
+	<div style="width:200px;height:40px;border-radius:5px;background-color:#BABFC6;color:black">
+		<div style="padding-top:10px"><a style="text-decoration:none;color:black" href="https://hub.docker.com/r/mlpack/singleuser">Docker Jupyter</a></div>
+	</div>
+
+	<div></div>
+</div>
+<!-- QUICKSTART-CLOUD END -->
+
+<br>
+<br>
+
+<!-- PACKAGE-GRID START -->
+<div class="grid-package">
+	<div id="f0"></div>
+
+	<div style="width:200px;color:black">Build</div>
+	<div id="build0" onclick="BuildSelect(0)" style="width:250px;background-color:rgba(208,51,31,0.9);color:white">Stable</div>
+	<div id="build1" onclick="BuildSelect(1)" style="width:250px;background-color:rgba(31,54,77,0.3);color:black">Nightly</div>
```

Comment:
Would "nightly" really be the right word here?  The instructions would be just for the git master branch, which technically isn't nightly, it's anytime.  Maybe "unstable" or "development"?  Not sure what the best term would be.

Reply:
Good point, `development ` sounds good to me.